packages/components/pages/f-contact-preferences/src/components/_tests/ContactPreferences.test.js

```diff
@@ -100,24 +100,23 @@ describe('ContactPreferences Component', () => {
     });
 
     afterEach(() => {
-        initialiseSpy = null;
         jest.clearAllMocks();
     });
 
     describe('when creating the component', () => {
-        it('should register the Store Module', () => {
+        it('should register the Store Module', async () => {
             // Arrange & Act
-            wrapper = mountContactPreferences({ storeOverride: new Vuex.Store() });
+            wrapper = await mountContactPreferences();
 
             // Assert
-            expect(registerStoreModuleSpy).toHaveBeenCalled();
+            expect(wrapper.vm.$store.state.fContactPreferencesModule).toBeDefined();
```

Comment:
Does this prove that our component registered the module?

Reply:
No, good catch. Not sure why I removed it.